sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
```

Comment:
yes, you have shown here that there are two entries for vti0 (one is prefixed with "ip_"), and yet they give different results.  That is why we must understand what each entry is telling us, and only check for the proper entry.